### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by HubertKwiatkowski**
I forgot to pull the main branch before branching to the avatar, that's why there are changes in different files - this was the quickest way to clean it up (probably not the best); I will be more careful next time.

Also, I'm not sure about this:
image
looks weird, but maybe it's just me :D
it was the only way I could style the avatar size to show the proper size...

### CODE REVIEW

File: src/dataDisplay/Avatar/Avatar.module.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #FFFFFF;
+  margin: 0;
+}
+
+.avatarBig {
+  max-width: 40px;
```
Comment (Sumick) [inline src/dataDisplay/Avatar/Avatar.module.css:32]: **nitpick:** This element is so small that you can easily set it to fixed `width`, and `height`. No need to set maxWidth, and maxHeight

File: src/dataDisplay/Avatar/Avatar.module.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  width: 40px;
+  max-height: 40px;
+  height: 40px;
+}
+
+.avatarMedium {
+  max-width: 32px;
```
Comment (Sumick) [inline src/dataDisplay/Avatar/Avatar.module.css:39]: **nitpick:** Same as above

File: src/dataDisplay/Avatar/Avatar.module.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  width: 30px;
+  max-height: 32px;
+  height: 32px;
+}
+
+.avatarSmall {
+  max-width: 24px;
```
Comment (Sumick) [inline src/dataDisplay/Avatar/Avatar.module.css:46]: **nitpick:** Same as above

File: src/dataDisplay/Avatar/Avatar.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+interface AvatarProps {
+  src: string;
+  alt: string;
+  size?: "big" | "medium" | "small";
+}
+
+export const Avatar = ({
+  src = "",
+  alt = "",
```
Comment (Sumick) [inline src/dataDisplay/Avatar/Avatar.tsx:14]: **nitpick:** No need to set an empty string here

**Review by Sumick — APPROVED** [review]
Good job 👌 Leaving you few cleanup comments

File: src/dataDisplay/Avatar/Avatar.module.css (hunk 3 above)
Comment (HubertKwiatkowski) [reply src/dataDisplay/Avatar/Avatar.module.css:46]: hm... I had some troubles with sizes at some point and didn't check this after changing part of the HTML structure, now it's redundant

**Pull request merged**